src/backend/src/services/teams.services.ts

```diff
@@ -20,9 +20,12 @@ export default class TeamsService {
    * @param organizationId The organization the user is currently in
    * @returns a list of teams
    */
-  static async getAllTeams(organizationId: string): Promise<Team[]> {
+  static async getAllTeams(organizationId: string, onlyArchive: boolean): Promise<Team[]> {
```

Comment:
Could you break this into 2  methods? As it is, there are two contexts in which this method is used, and neither are actually to get all teams.

It may even be easier to make the differentiation in the frontend so you could have one endpoint and not have to combine the lists when youre displaying the admin tools table